src/library.cpp

```diff
@@ -157,6 +157,90 @@ bool Library::removeBookmark(const std::string& zimId, const std::string& url)
   return false;
 }
 
+int Library::migrateBookmarks() {
```

Comment:
I wonder if this function should return not only the number of migrated bookmarks, but also the count of those bookmarks that failed to migrate

Reply:
Yes